### PR TITLE
Tests for api

### PR DESCRIPTION
**Pull request by Masclins**
Joining `api/base.clj` tests and removing already considered tests.

### CODE REVIEW

File: backend/src/rules/count_cards.clj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (ns rules.count-cards)
 
 (defn count-cards
-  "Count all cards that has keys as in condition an sums their summand"
+  "Count all cards that has keys as in condition (or sums their summand)"
```
Comment (kenan-rhoton) [inline backend/src/rules/count_cards.clj]: *have* keys 🙃

File: backend/test/api/base_test.clj [hunk 2]
```diff
@@ -1,13 +1,18 @@
-(ns api.join-game-test
+(ns api.base-test
   (:require [expectations.clojure.test :refer :all]
             [clojure.test :as ctest]
             [mocking :as mocking]
             [api.base :as base]
-            [configs.hands :as hands]
```
Comment (kenan-rhoton) [inline backend/test/api/base_test.clj]: Wait, this is used below, right? as `hands/default-hand`?

**Review by kenan-rhoton — DISMISSED** [review]

File: backend/test/api/base_test.clj (hunk 2 above)
Comment (Masclins) [reply backend/test/api/base_test.clj]: Probably that's why it fails the tests (sorry, couldn't run them).
Still... I don't understand how I removed this line... Thanks.

**Review by kenan-rhoton — APPROVED** [review]
This is just a cleanup + reorganization, so not much to review. Approved!

**Pull request merged**